dmutils/formats.py

```diff
@@ -181,20 +183,28 @@ def _use_gmt_timezone(date_string):
     return re.sub(r"UTC|BST", "GMT", date_string)
 
 
-def _format_date(value, default_value, fmt, localize=True):
-    if not value:
-        return default_value
-
+def get_datetime(value: Union[str, datetime], localize: bool = True) -> datetime:
```

Comment:
Could we call this `get_localized_datetime`?

Reply:
Done